pkg/rc/fields/fields.go

```diff
@@ -49,6 +52,10 @@ type RC struct {
 
 	// When disabled, this controller will not make any scheduling changes
 	Disabled bool
+
+	// Distinguishes between dynamic, static or other strategies for allocating
+	// nodes on which the rc can schedule the manifest.
+	AllocationStrategy Strategy
```

Comment:
you're also going to need to add this to `RawRC` and probably copy the value to/from the RC in the JSON marshaling functions. Sorry this stuff is so convoluted, i think it doesn't have to be but that's how it is :)